Kickstarter-iOS/Features/ConfirmDetails/Controllers/ConfirmDetailsViewController.swift

```diff
@@ -0,0 +1,282 @@
+
+  func configure(with data: PledgeViewData) {
+    self.viewModel.inputs.configure(with: data)
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    self.navigationItem
+      .backBarButtonItem = UIBarButtonItem(title: " ", style: .plain, target: nil, action: nil)
+
+    _ = self
+      |> \.extendedLayoutIncludesOpaqueBars .~ true
+
+    _ = self
+      |> \.title .~ Strings.Back_this_project()
+
+    self.view.addGestureRecognizer(self.keyboardDimissingTapGestureRecognizer)
+
+    self.configureChildViewControllers()
+    self.setupConstraints()
+
+    self.viewModel.inputs.viewDidLoad()
+  }
+
+  // MARK: - Configuration
+
+  private func configureChildViewControllers() {
+    _ = (self.rootScrollView, self.view)
+      |> ksr_addSubviewToParent()
+
+    _ = (self.rootStackView, self.rootScrollView)
+      |> ksr_addSubviewToParent()
+      |> ksr_constrainViewToEdgesInParent()
+
+    let childViewControllers = [
+      self.pledgeAmountViewController,
+      self.shippingLocationViewController
+    ]
+
+    let arrangedSubviews = [
+      self.rootInsetStackView
+    ]
+
+    let arrangedInsetSubviews = [
+      [self.titleLabel],
+      self.inputsSectionViews
+    ]
+    .flatMap { $0 }
+    .compact()
+
+    arrangedSubviews.forEach { view in
+      self.rootStackView.addArrangedSubview(view)
+    }
+
+    arrangedInsetSubviews.forEach { view in
+      self.rootInsetStackView.addArrangedSubview(view)
+    }
+
+    childViewControllers.forEach { viewController in
+      self.addChild(viewController)
+      viewController.didMove(toParent: self)
+    }
+  }
+
+  private func setupConstraints() {
+    NSLayoutConstraint.activate([
+      self.rootScrollView.topAnchor.constraint(equalTo: self.view.topAnchor),
+      self.rootScrollView.leftAnchor.constraint(equalTo: self.view.leftAnchor),
+      self.rootScrollView.rightAnchor.constraint(equalTo: self.view.rightAnchor),
+      self.rootScrollView.bottomAnchor.constraint(equalTo: self.view.bottomAnchor),
+      self.rootStackView.widthAnchor.constraint(equalTo: self.view.widthAnchor)
+    ])
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.view
+      |> checkoutBackgroundStyle
+
+    _ = self.titleLabel
+      |> titleLabelStyle
+
+    _ = self.rootScrollView
+      |> rootScrollViewStyle
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.rootInsetStackView
+      |> rootInsetStackViewStyle
+  }
+
+  // MARK: - View model
+
+  override func bindViewModel() {
+    super.bindViewModel()
+
+    self.viewModel.outputs.configureLocalPickupViewWithData
+      .observeForUI()
+      .observeValues { [weak self] data in
+        self?.localPickupLocationView.configure(with: data)
+      }
+
+    self.viewModel.outputs.configureShippingLocationViewWithData
+      .observeForUI()
+      .observeValues { [weak self] data in
+        self?.shippingLocationViewController.configureWith(value: data)
```

Comment:
Nit: the other methods are `configure(with:)` and this is `configureWith(value:)`